modules/_labs/drawer/react/lib/DrawerHeader.tsx

```diff
@@ -97,12 +97,14 @@ export default class DrawerHeader extends React.Component<DrawerHeaderProps, {}>
         <HeaderTitle id={id} inverse={inverse} title={title}>
           {title}
         </HeaderTitle>
-        <CloseButton
-          variant={inverse ? IconButtonVariant.Inverse : IconButtonVariant.Plain}
-          onClick={onClose}
-          aria-label={closeIconLabel}
-          icon={xIcon}
-        ></CloseButton>
+        {onClose && closeIconLabel ? (
```

Comment:
Nit: You can use short circuiting to simplify this: `{onClose && closeIconLabel && <CloseButton ... />}`